datalad/support/annexrepo.py

```diff
@@ -12,23 +12,53 @@
 
 """
 
-from os.path import join, exists
+from os import getcwd
+from os.path import join as p_join, exists, normpath, isabs, commonprefix, relpath
```

Comment:
I'd prefer a name, that clearly states what join I'm using. It's mostly a
matter of taste, of course, but even if there is no current collision, I
still feel better by a reduced potential of collision.

2015-04-12 17:43 GMT+02:00 yarikoptic-private <notifications@github.com>:

> In datalad/support/annexrepo.py
> <https://github.com/datalad/datalad/pull/90#discussion_r28204472>:
>
> > @@ -12,23 +12,53 @@
> >
> >  """
> >
> > -from os.path import join, exists
> > +from os import getcwd
> > +from os.path import join as p_join, exists, normpath, isabs, commonprefix, relpath
>
> we should unify how we import this path.join... I did use pjoin at some
> point, simple join at other spots, then converged to use "opj" as testkraut
> uses... p_join becomes fourth one I believe. IMHO -- should be either
> simple join or opj -- which one? I would vote now for a simple join -- it
> is unlikely to collide with anything.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datalad/datalad/pull/90/files#r28204472>.
>